apis/dataflow/v1beta1/dataflowflextemplatejob_types.go

```diff
@@ -0,0 +1,459 @@
+		Environment *FlexTemplateRuntimeEnvironment `json:"environment,omitempty"`
+
+		// Set this to true if you are sending a request to update a running
+		//  streaming job. When set, the job name should be the same as the
+		//  running job.
+		Update *bool `json:"update,omitempty"`
+
+	*/
+
+	// The initial number of Google Compute Engine instances for the job.
+	NumWorkers *int32 `json:"numWorkers,omitempty"`
+
+	// The maximum number of Google Compute Engine instances to be made
+	//  available to your pipeline during execution, from 1 to 1000.
+	MaxWorkers *int32 `json:"maxWorkers,omitempty"`
+
+	/* NOTYET
+	// The Compute Engine [availability
+	//  zone](https://cloud.google.com/compute/docs/regions-zones/regions-zones)
+	//  for launching worker instances to run your pipeline.
+	//  In the future, worker_zone will take precedence.
+	Zone *string `json:"zone,omitempty"`
+	*/
+
+	// The email address of the service account to run the job as.
+	ServiceAccountEmailRef *refs.IAMServiceAccountRef `json:"serviceAccountEmailRef,omitempty"`
+
+	// The Cloud Storage path to use for temporary files.
+	//  Must be a valid Cloud Storage URL, beginning with `gs://`.
+	TempLocation *string `json:"tempLocation,omitempty"`
+
+	// The machine type to use for the job. Defaults to the value from the
+	//  template if not specified.
+	MachineType *string `json:"machineType,omitempty"`
+
+	// Additional experiment flags for the job.
+	AdditionalExperiments []string `json:"additionalExperiments,omitempty"`
+
+	// Network to which VMs will be assigned.  If empty or unspecified,
+	//  the service will use the network "default".
+	NetworkRef *refs.ComputeNetworkRef `json:"networkRef,omitempty"`
+
+	// Subnetwork to which VMs will be assigned, if desired. You can specify a
+	//  subnetwork using either a complete URL or an abbreviated path. Expected to
+	//  be of the form
+	//  "https://www.googleapis.com/compute/v1/projects/HOST_PROJECT_ID/regions/REGION/subnetworks/SUBNETWORK"
+	//  or "regions/REGION/subnetworks/SUBNETWORK". If the subnetwork is located in
+	//  a Shared VPC network, you must use the complete URL.
+	SubnetworkRef *refs.ComputeSubnetworkRef `json:"subnetworkRef,omitempty"`
+
+	/* NOTYET
+	// Additional user labels to be specified for the job.
+	//  Keys and values must follow the restrictions specified in the [labeling
+	//  restrictions](https://cloud.google.com/compute/docs/labeling-resources#restrictions)
+	//  page.
+	//  An object containing a list of "key": value pairs.
+	//  Example: { "name": "wrench", "mass": "1kg", "count": "3" }.
+	AdditionalUserLabels map[string]string `json:"additionalUserLabels,omitempty"`
+	*/
+
+	// The Cloud KMS key for the job.
+	//  Key format is:
+	//  projects/<project>/locations/<location>/keyRings/<keyring>/cryptoKeys/<key>
+	KmsKeyNameRef *refs.KMSCryptoKeyRef `json:"kmsKeyNameRef,omitempty"`
+
+	// Configuration for VM IPs.
+	IpConfiguration *string `json:"ipConfiguration,omitempty"`
```

Comment:
nit: `IPConfiguration`

Reply:
Good call.  I think it wasn't caught by the apichecker because that checks the JSON form (via the CRD)